sdcm/tester.py

```diff
@@ -378,16 +378,20 @@ def __init__(self, *args, **kwargs):  # pylint: disable=too-many-statements,too-
     def init_argus_run(self):
         try:
             self.test_config.init_argus_client(self.params)
-            self.test_config.argus_client().submit_sct_run(
-                job_name=get_job_name(),
-                job_url=get_job_url(),
-                started_by=get_username(),
-                commit_id=get_git_commit_id(),
-                runner_public_ip=get_sct_runner_ip(),
-                runner_private_ip=get_my_ip(),
-                sct_config=self.params,
-            )
-            self.log.info("Initialized Argus TestRun with test id %s", self.test_config.argus_client().run_id)
+            if self.params.get('argus_test_id') != self.test_config.test_id():
```

Comment:
Shouldn't we also update the `self.params.get('argus_test_id')` value for case we call this method second+ time?

Also, cannot we just call Argus API to get to know whether there is an existing Argus test run with this ID or not?
If we can, won't it drop the need to introduce the new config option?